upload-cloud-storage/src/client.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import { UploadHelper } from './upload-helper';
+import { Storage, UploadResponse } from '@google-cloud/storage';
+
+/**
+ * Available options to create the client.
+ *
+ * @param credentials GCP JSON credentials (default uses ADC).
+ */
+
+type ClientOptions = {
+  credentials?: string;
+};
+
+/**
+ * Handles credential lookup, registration and
+ * wraps interactions with the GCS Helper
```

Comment:
Missing trailing '.'
Also line wrapping is inconsistent.

Reply:
Done, comments line-wrap at 80 now.